charts/private-action-runner/README.md

```diff
@@ -1,9 +1,18 @@
 # Datadog Private Action Runner
 
-![Version: 0.6.0](https://img.shields.io/badge/Version-0.6.0-informational?style=flat-square) ![AppVersion: v0.0.1-alpha27](https://img.shields.io/badge/AppVersion-v0.0.1--alpha27-informational?style=flat-square)
+![Version: 0.7.0](https://img.shields.io/badge/Version-0.7.0-informational?style=flat-square) ![AppVersion: v0.0.1-alpha27](https://img.shields.io/badge/AppVersion-v0.0.1--alpha27-informational?style=flat-square)
 
 This Helm Chart deploys the Datadog Private Action runner inside a Kubernetes cluster. It allows you to use private actions from the Datadog Workflow and Datadog App Builder products. When deploying this chart, you can give permissions to the runner in order to be able to run Kubernetes actions.
 
+## How to use Datadog Helm repository
+
+You need to add this repository to your Helm repositories:
```

Comment:
you mentioned yesterday that this might not work since the chart needs to be modified with the runner's generated config values. we should test that and make sure the instructions reflect that usage.

Reply:
It will work with the config, just not with connection credentials which I need to figure out a better experience for so that it works with this

Reply:
got it ok.

i noticed that this PR also deleted the instructions for how to update the values with the config. i think we should have that somewhere cause otherwise customers won't know to do it. we can include the `--enroll-and-print-config` flag too!

Reply:
So we put those instructions in the new UI:
<img width="1216" alt="image" src="https://github.com/user-attachments/assets/d916bb6b-acbb-4638-a216-6fe5747b31e2">
it's live now on prod https://app.datadoghq.com/app-builder/private-action-runners/new
I can copy them over to the chart if it'd be good to have the instructions in both places?

Reply:
talked offline, this is good 👍